include/z64player.h

```diff
@@ -317,6 +317,13 @@ typedef enum PlayerDoorType {
     /*  5 */ PLAYER_DOORTYPE_PROXIMITY
 } PlayerDoorType;
 
+// Some player animations are played at this reduced speed, for reasons yet unclear.
+// Perhaps to compress animation data? Or to adjust from 30fps to 20fps?
+// This is called "adjusted" for now.
+// z_en_horse also has many instances of this adjusted speed
+#define PLAYER_ANIM_ADJUSTED_SPEED (2.0f / 3.0f)
+#define PLAYER_ANIM_NORMAL_SPEED   (3.0f / 3.0f)
```

Comment:
I'm not super sold on this, I think introducing this we'd expect it to multiply the spots where a dynamic speed is used (for example the running/walking animation speed iirc), but that would probably not match

i.e. where `AnimChange(speed = calculatedSpeed)`, `AnimChange(speed = calculatedSpeed * ANIMSPEEDNORMAL)` wouldn't match

Reply:
Are dynamic speeds multiplied by `PLAYER_ANIM_ADJUSTED_SPEED` in any case?

Reply:
I suppose there is `PLAYER_ANIM_ADJUSTED_SPEED * sPlayerWaterSpeedScale`

Reply:
Maybe your `calculatedSpeed` was already multiplied by one of those two defines? i.e `calculatedSpeed = someSpeed * PLAYER_ANIM_NORMAL_SPEED`

Do you have a more concrete example?

Reply:
Yeah, can confirm that doing `* PLAYER_ANIM_NORMAL_SPEED` i.e. `* 1.0f` doesn't match in a lot of places. It's more the exception where this needs to be applied more, so it's worth revisiting once player docs are further along